asf_public_discourse_home_decarbonisation/getters/mse_getters.py

```diff
@@ -0,0 +1,113 @@
+"""
+Getter functions for MSE data.
+"""
+import pandas as pd
+from asf_public_discourse_home_decarbonisation.getters.getter_utils import load_s3_data
+from asf_public_discourse_home_decarbonisation import PROJECT_DIR
+import sys
+import logging
+
+logger = logging.getLogger(__name__)
+
+S3_BUCKET = "asf-public-discourse-home-decarbonisation"
+PATH_TO_LOCAL_MSE_DATA = PROJECT_DIR / "inputs/data/mse"
+
+
+def get_first_attempt_mse_data() -> pd.DataFrame:
+    """
+    Returns a dataframe with 10 pages of MSE data from the Green Ethical Money Saving category.
+    This was the first sample of data collected, used to generate the first insights.
+    """
+    return load_s3_data(
+        bucket_name=S3_BUCKET,
+        file_path="data/mse/first_attempt/green-ethical-moneysaving.parquet",
+    )
+
+
+def get_mse_category_data(
+    category: str, collection_date: str, read_from_local_path: bool = False
+) -> pd.DataFrame:
+    """
+    Returns a dataframe with most up to date category/sub-forum data
```

Comment:
Hummm this needs an update. I think I had something different in mind originally - getting the most up to date batch of data, instead of data collected in a specific date. But each batch basically contains all historical data up to the collection date.

Reply:
Fixed. :)